package.json

```diff
@@ -5,6 +5,7 @@
   "repository": "unjs/jiti",
   "license": "MIT",
   "main": "./lib/index.js",
+  "module": "./lib/index.mjs",
```

Comment:
Currently, little bit hesitated to introduce it as `index.mjs` which is dedicated to fallback (test, browser) environments. It makes behavior partial and less consistent + the fact  that `module` is not a standard format and (test, browser) builders can also pick `exports` field (which we should add) and it can again break the functionality of what we add.

Reply:
Yeah, but not sure how we could do it otherwise (Vite throws when importing CJS). I guess maybe we could release a patch with `jiti.import`, update `unbuild` with `await jiti.import()`. And then have this as a major-beta. So users could opt-in this by override with the beta version of jiti.

I know quite a few ppl waiting for this capability for both frontend and tests

Reply:
Seems a good plan for unbuild. I already wanted to start these as new major version.